ServerCore/Pages/Puzzles/Create.cshtml.cs

```diff
@@ -31,6 +31,7 @@ public IActionResult OnGet()
 
         public async Task<IActionResult> OnPostAsync()
         {
+            ModelState.Remove("Puzzle.Event");
```

Comment:
I see it in many places but its meaning isn't clear to me.

Reply:
It tells the page not to validate the Event property -- as submitted, it's invalid because it's null (and thus a violation of ```[Required]```. We set it further down before sending it to the database. There are some similar cases on Edit pages where we just don't roundtrip a required property (usually the DB just don't get changed for those).